node/engine/messageservice/p2p-message-service/service.go

```diff
@@ -53,8 +53,9 @@ const (
 
 // P2PMessageService is a rudimentary message service that uses TCP to send and receive messages.
 type P2PMessageService struct {
-	toEngine chan protocols.Message // for forwarding processed messages to the engine
-	peers    *safesync.Map[peer.ID]
+	InitComplete chan bool
```

Comment:
This might be nitpicking but we could use a private channel and expose it via a public getter. The advantage of that is we can expose `InitComplete` as a one-way read only chan to public consumers.  [Here's an example from the chain service.](https://github.com/statechannels/go-nitro/blob/18d7bd40d9ce4af0abaa2317d0e5a7ebb3b61bfe/node/engine/chainservice/eth_chainservice.go#L305)

Reply:
Done in https://github.com/statechannels/go-nitro/pull/1515/commits/ecfd7dfb5ea14f594288e01681f05f3e50c71f76